cinema/serializers.py

```diff
@@ -132,6 +149,31 @@ def create(self, validated_data):
                 Ticket.objects.create(order=order, **ticket_data)
             return order
 
+    def update(self, instance, validated_data):
+        tickets_data = validated_data.pop("tickets")
```

Comment:
The `update` method in `OrderSerializer` should handle the case where `tickets` might not be present in `validated_data`. Consider adding a check to ensure `tickets_data` is only processed if it exists in `validated_data`.